Content.Server/Drone/DroneSystem.cs

```diff
@@ -130,5 +134,19 @@ private void UpdateDroneAppearance(EntityUid uid, DroneStatus status)
                 appearance.SetData(DroneVisuals.Status, status);
             }
         }
+
+        private bool NonDronesInRange(EntityUid uid, DroneComponent component)
+        {
+            var xform = Comp<TransformComponent>(uid);
+            foreach (var entity in _lookup.GetEntitiesInRange(xform.MapID, xform.WorldPosition, component.InteractionBlockRange))
+            {
+                if (HasComp<MindComponent>(entity) && !HasComp<DroneComponent>(entity) && !HasComp<GhostComponent>(entity))
```

Comment:
I think it would make sense to add a `DroneUsable` tag check here? Or maybe some other mob-check specific tag? That way future mobs/silicon that should be drone-interactable could just get given the tag, instead of having to update this function. I guess that could also replace the `HasComp<DroneComponent>` check.

Reply:
Drones can't interact with silicons either

Reply:
I guess its fine as it is then.